src/client/automation/playback/click/select-child.js

```diff
@@ -159,9 +151,16 @@ export default class SelectChildClickAutomation {
                 eventSimulator.change(this.parentSelect);
             }
         }
-        else
+        else {
             eventSimulator.mouseup(this.eventsArgs.element, this.eventsArgs.options);
 
+            if ((browserUtils.isFirefox || browserUtils.isSafari ||
+                 browserUtils.isChrome && browserUtils.version >= 53) && this.clickCausesChange) {
+                eventSimulator.input(this.parentSelect);
+                eventSimulator.change(this.parentSelect);
+            }
+        }
+
```

Comment:
I decide to stay on @AndreyBelym remark, cause in this case it will be look next:
```js:
_mouseup () {
        var elementForMouseupEvent = browserUtils.isIE ? this.parentSelect : this.eventsArgs.element;

        eventSimulator.mouseup(elementForMouseupEvent, this.eventsArgs.options);

        if (browserUtils.isIE && this.clickCausesChange)
            this.parentSelect.selectedIndex = this.childIndex;

        var unexplainedCondition = browserUtils.isFirefox || browserUtils.isSafari ||
                                           browserUtils.isChrome && browserUtils.version >= 53;

        var shouldSimulateInput = unexplainedCondition && this.clickCausesChange;

        var shouldSimulateChange = (unexplainedCondition || browserUtils.isIE) && this.clickCausesChange;

        if (shouldSimulateInput)
            eventSimulator.input(this.parentSelect);

        if (shouldSimulateChange)
            eventSimulator.change(this.parentSelect);

        return Promise.resolve();
    }
```
Only if you can suggest a meaningfull name for `unexplainedCondition` variable